airgun/entities/host.py

```diff
@@ -210,14 +210,25 @@ def set_puppet_class_parameter_value(self, entity_name, name, value):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
-    def open_webconsole(self, entity_name):
-        """Navigate to host's webconsole
+    def has_working_webconsole(self, entity_name):
+        """Navigate to host's webconsole and return the hostname from the cockpit page
```

Comment:
Instead changing the whole use case of this function, I would have added an optional parameter to return the view. @mirekdlugosz, @tstrych  what do you think about it?

Reply:
Yes, function with that name should return boolean value.
No, I don't think optional parameter to return view would be appropriate. So far we have managed without exposing views to robottelo, I would like to try to keep it this way.

This should be named something like `get_webconsole_content()`, and do:
1. navigate to specific page
2. switch frames
3. get content of `webconsole_view` (`wait_for` should return it on success, just wrapped in some structure)
4. switch frames back
5. Return content from step 3